x-pack/plugins/siem/public/components/charts/common.tsx

```diff
@@ -79,3 +85,24 @@ export const getSeriesStyle = (
 
   return customSeriesColors;
 };
+
+export const getTheme = () => {
+  const theme: PartialTheme = {
+    chartMargins: {
+      left: 0,
+      right: 0,
+      top: 4,
```

Comment:
Apply 4px margin on top to fix the problem of chopping first y-tick
<img width="356" alt="Screenshot 2019-06-01 at 01 26 26" src="https://user-images.githubusercontent.com/6295984/58723309-501c0b80-840c-11e9-8124-902024f47702.png">